### PR TITLE
GHC 8.8 compatibility

### PR DESCRIPTION
**Pull request by fosskers**
Just one little bounds widening.

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -1,4 +1 @@
-resolver: lts-13.7
-
-packages:
-- .
+resolver: lts-14.7
```
Comment (fosskers) [inline stack.yaml]: Note that this is still an 8.6 LTS. There isn't yet one for 8.8.

**Comment by fosskers** [comment]
Hi there, any thoughts on this?

**Comment by fosskers** [comment]
Hi there, any updates?

**Comment by sigrlami** [comment]
Hi, Colin.Sorry for long reply, I'll check everything this week and test compatibility for issues. On Dec 17, 2019 23:39, Colin Woodbury <notifications@github.com> wrote:Hi there, any updates?

**Comment by fosskers** [comment]
Not a problem, cheers.

**Comment by fosskers** [comment]
Hi there, any progress?

**Comment by sigrlami** [comment]
@fosskers merged. Hackage release will be published a bit later, need to finish with tests

**Comment by fosskers** [comment]
Wonderful, thank you very much!

**Pull request merged**